edge_computing/ztp-sno-additional-worker-node.adoc

```diff
@@ -28,6 +28,8 @@ include::snippets/technology-preview.adoc[]
 
 * For more information about worker nodes, see xref:../nodes/nodes/nodes-sno-worker-nodes.adoc#nodes-sno-worker-nodes[Adding worker nodes to {sno} clusters].
 
+* For information about removing a worker node from an expanded {sno-caps} cluster, see link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.10/html/clusters/cluster_mce_overview#auto-remove-host-steps-cli[Removing managed cluster nodes by using the command line interface].
```

Comment:
I'm wondering why in the new bullet point you use the attribute for `Single-node OpenShift cluster`, but in the first bullet point in the same list, it's `single-node OpenShift clusters`. What's the difference? Should both be using the same attribute?

![image](https://github.com/openshift/openshift-docs/assets/86380613/7bf21581-5eb7-48e5-89fe-478876f61571)

Reply:
Yes, you're right - I didn't notice that. There are two different attributes, so I'll change the first one. Thanks!